settings.gradle

```diff
@@ -2,6 +2,7 @@ rootProject.name = 'conjure-java-runtime'
 
 include 'conjure-java-annotations'
 include 'conjure-java-jaxrs-client'
+include 'conjure-java-jaxrs-jakarta-client'
```

Comment:
I'm not sure that we can get away with branching conjure-java-jaxrs-clients given the migration won't be typesafe (input is a `Class<?>`). I've meant to look into replacing our feign contract implementation to support both sets of annotations so that we can support existing interfaces more easily.